src/EnergyPlus/InputProcessing/IdfParser.cc

```diff
@@ -228,16 +228,6 @@ json IdfParser::parse_idf(std::string const &idf, size_t &index, bool &success,
                 while (token != Token::SEMICOLON && token != Token::END)
                     token = next_token(idf, index);
                 continue;
-            } else if (obj_name.find("Parametric:") != std::string::npos) {
```

Comment:
Move checks for unsupported objects out of `parse_idf` so the same checks can apply for idf and epJSON input.